integration_tests/samples/socket_mode/bolt_adapter/aiohttp.py

```diff
@@ -0,0 +1,107 @@
+# ------------------
+# Only for running this script here
+import sys
+from os.path import dirname
+
+sys.path.insert(1, f"{dirname(__file__)}/../../../..")
+# ------------------
+
+import json
+import logging
+import os
+import asyncio
+from typing import Optional
+from time import time
+from slack_sdk.socket_mode.response import SocketModeResponse
+from slack_sdk.socket_mode.request import SocketModeRequest
+from slack_sdk.socket_mode.aiohttp import SocketModeClient
+
+from slack_bolt import App
+from slack_bolt.request import BoltRequest
+from slack_bolt.response import BoltResponse
+
+
+class SocketModeAdapter:
+    app: App
+
+    def __init__(self, app: App):
+        self.app = app
+        # TODO: remove this once Bolt allow skipping this
+        if self.app._signing_secret is None:
+            self.app._signing_secret = "just-for-workaround"
+        from slack_sdk.signature import SignatureVerifier
+
+        self.signature_verifier = SignatureVerifier(
+            signing_secret=self.app._signing_secret
+        )
```

Comment:
these lines will be removed once https://github.com/slackapi/bolt-python/pull/154 is shipped